configure.py

```diff
@@ -142,6 +142,8 @@ def standard_supported(standard, compiler='g++'):
 arg_parser.add_argument('--dpdk-machine', default='native', help='Specify the target architecture')
 add_tristate(arg_parser, name='deferred-action-require-noexcept', dest='deferred_action_require_noexcept', help='noexcept requirement for deferred actions', default=True)
 arg_parser.add_argument('--prefix', dest='install_prefix', default='/usr/local', help='Root installation path of Seastar files')
+arg_parser.add_argument('--sanitizer', dest='sanitizers', action='append', default=[], help='Use specified sanitizer')
+arg_parser.add_argument('--no-sanitizers', dest='no_sanitizers', action='store_true', default=False, help='Do not use sanitizers')
```

Comment:
after a second thought, i think probably it's simpler to accept a colon-separated or a comma-separated list as the argument as `--sanitizers`? like

```py
g_parser.add_argument('--sanitizers', dest='sanitizers',  default='address;undefined_behavior', help='Use specified sanitizers')
```
what do you think?

Reply:
done